src/main/preferences/index.js

```diff
@@ -41,8 +33,10 @@ class Preference extends EventEmitter {
 
     const { userDataPath } = paths
     this._userDataPath = userDataPath
+    this.hasPreferencesFile = fs.existsSync(path.join(this._userDataPath, `./${PREFERENCES_FILE_NAME}.json`))
     this.store = new Store({
-      schema
+      schema,
+      name: PREFERENCES_FILE_NAME
```

Comment:
Please set `cwd` to `preferencesPath` to support later changes.

Reply:
cwd is a relative path, do we really need to set the path? I think store it in userData folder is ok?

Reply:
No thats wrong, it's an absolute path and I think it should be set because with session I'll move the electron data like cache etc into a subdirectory. There is so much trash in the configuration folder.

Reply:
Ok, can you update it later when you add session.